.gitignore

```diff
@@ -0,0 +1,2 @@
+node_modules/
+coverage
```

Comment:
- [ ] Again here as requested by the previous reviewer, the project requirements state that you should create a `test.md` file in your project and you should add it to your `,gitignore` file so that when you push to GitHub, the file will not be pushed but rather only a link to the file will remain in the `.gitignore` file to show that it was excluded. Kindly do so